lalrpop/src/grammar/repr.rs

```diff
@@ -109,6 +109,7 @@ pub struct Production {
 pub enum Symbol {
     Nonterminal(NonterminalString),
     Terminal(TerminalString),
+    Error,
```

Comment:
Is this the correct placement of `error`. Considered putting it under `Symbol::Terminal` as well but this was simpler (barring the current travis failure).